pages/blog/asyncapi-discovery-intro.md

```diff
@@ -0,0 +1,75 @@
+
+Oftentimes, removing an event topic or queue requires nerves of steel, for fear it might disrupt key functionality. Many a middleware engineer has found religion during a production broker clean-up.
+
+Thankfully, the AsyncAPI Discovery Tool offers a better way.
+# How the AsyncAPI Discovery Tool Works
+The AsyncAPI Discovery Tool analyzes event traffic passing through brokers like Kafka, RabbitMQ, IBM MQ, Solace, and more. After learning how the broker distributes events, the AsyncAPI Discovery Tool generates a corresponding [AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.2.0). The generated spec can be used for code generation, documentation, visualization, infrastructure deployment, and more.
+
+It’s a great starting point for getting events catalogued and governed.
+
+It’s not perfect (more on that later), and there’s a lot more work to be done, but the AsyncAPI Discovery Tool helps your enterprise align production reality with AsyncAPI documentation. And perhaps relieve some tension for middleware engineers.
+
+# Getting Started with AsyncAPI Discovery Tool
+Getting started means a trip to the [SolaceLabs GitHub](https://github.com/SolaceLabs/event-discovery-agent), where you can find detailed instructions and documentation. (While Solace created AsyncAPI Discovery, it’s open source with an Apache 2.0 license.)
+
+The AsyncAPI Discovery Tool runs as a stand-alone Java Jar, so getting it running requires only Java and Maven. Once it’s up and running, AsyncAPI has its own self-contained UI, offering fill-in-the-blanks configuration ([get more details about the UI](https://github.com/SolaceLabs/event-discovery-agent/blob/main/docs/ui.md)). Here’s an example for Kafka:
+
+![Figure 1: UI for AsyncAPI Discovery for Kafka](/img/posts/asyncapi-discovery-tool/asyncapi-discovery-tool-1.webp)
+
+Just fill in the configuration, asking your friendly local administrator for help if needed, and then click the “Start Scan” button. After grinding away, the AsyncAPI Discovery Tool returns a consolidated spec file. The AsyncAPI file describes the channels and schemas of events passing through the broker. From there, the world is your oyster: generate code, create infrastructure, or start governing your events.
+
+![Figure 2: Resulting (greatly simplified) AsyncAPI spec](/img/posts/asyncapi-discovery-tool/asyncapi-discovery-tool-2.webp)
+
+
+# The Future of the AsyncAPI Discovery Tool
+
+The most obvious place to improve the AsyncAPI Discovery Tool is to expand the number of supported brokers. Right now, it supports:
+
+- Apache Kafka
+- Solace PubSub+
+- NATS
+- RabbitMQ
+- HiveMQ
+
+Fortunately, the tool was built with extensibility in mind. There is a documented plug-in architecture just itching to have more brokers added. If you’re interested, The people who maintain the tool welcome pull requests.
+
+- The confusing publish and subscribe verbs in the output.
```

Comment:
I think these bullet points merit a better introduction. Going from your last paragraph to these bullet points felt choppy and slightly disorienting. :) 

Are you trying to explain that these are 2 examples of PRs you want the community to start? To continue with you? Or..?